packages/cli/src/args.ts

```diff
@@ -13,6 +13,7 @@ export const help = () => `
       deploy               [path]      Performs a deployment ${chalk.bold(
         '(default)'
       )}
+      build                            Build the project locally
```

Comment:
There will be a snapshot test that fails because of this change.

Reply:
@erikareads I oddly couldn't find the snapshot (or the test even) for `help/--help` outside the context of a specific command. Any idea where to look?

Reply:
I guess we'll see in a moment!

Reply:
Aha! `test/unit/args.test.ts`. I was just looking at `help.test.ts`